Libplanet.Action/ActionEvaluator.cs

```diff
@@ -93,19 +100,20 @@ public IReadOnlyList<IActionEvaluation> Evaluate(IPreEvaluationBlock block)
                     EvaluateBlock(block, previousState).ToImmutableList();
 
                 var policyBlockAction = _policyBlockActionGetter(block);
-                if (policyBlockAction is null)
-                {
-                    return evaluations;
-                }
-                else
+                if (!(policyBlockAction is null))
                 {
-                    previousState = evaluations.Count > 0
-                        ? evaluations.Last().OutputState
-                        : previousState;
-                    return evaluations.Add(
-                        EvaluatePolicyBlockAction(block, previousState)
-                    );
+                     previousState = evaluations.Count > 0
```

Comment:
Tab space of 5? 🙃

Reply:
https://github.com/planetarium/libplanet/pull/3392/commits/6bbed288b2f7566eb8a924a19c2963663be0d2d4 amended.